docs/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,743 @@
+= {project-name}: A _native_ EPUB3 converter for AsciiDoc
```

Comment:
The title of the index page should be:

```suggestion
= Asciidoctor EPUB3 Documentation
```

If you follow this convention, the breadcrumb will display: `Asciidoctor EPUB3 > Page Title`.
You should also add the attribute `:navtitle: Introduction`.

Reply:
Thanks, updated PR accordingly.